exhibitor-core/src/main/resources/com/netflix/exhibitor/core/ui/index.html

```diff
@@ -165,6 +165,7 @@
                         <legend class="caution-legend">Paths</legend>
                         <label for="config-zookeeper-install-dir">ZooKeeper Install Dir</label><input type="text" id="config-zookeeper-install-dir" name="config-zookeeper-install-dir" size="60" title="The directory where the ZooKeeper server is installed">
                             <button id="config-zookeeper-install-dir-help-button">Read Me</button><br clear="all"/>
+                        <label for="config-zookeeper-config-dir">ZooKeeper Configuration Dir</label><input type="text" id="config-zookeeper-config-dir" name="config-zookeeper-config-dir" size="60" title="The directory where the ZooKeeper config is stored. Leave blank to use the ZooKeeper Snapshot Dir"><br clear="all"/>
```

Comment:
I don't think so, the config directory has to have a value if it doesn't then we end up in the same scenario as the original bug.